worker/src/RTC/RTCP/XrDelaySinceLastRr.cpp

```diff
@@ -75,14 +75,12 @@ namespace RTC
 				{
 					report->AddSsrcInfo(ssrcInfo);
 					offset += ssrcInfo->GetSize();
+					reportLength -= ssrcInfo->GetSize();
 				}
 				else
 				{
 					return report.release();
 				}
-
-				offset += ssrcInfo->GetSize();
```

Comment:
This was the terrible bug: this was duplicated (see line 77 above) so offset was decremented twice.